project/SparkBuild.scala

```diff
@@ -1104,7 +1104,15 @@ object CopyDependencies {
 
 object TestSettings {
   import BuildCommons._
-  private val defaultExcludedTags = Seq("org.apache.spark.tags.ChromeUITest")
+  private val isMacOnAppleSilicon = System.getProperty("os.name").startsWith("Mac OS X") &&
+    System.getProperty("os.arch").equals("aarch64")
+  private val defaultExcludedTags = if (isMacOnAppleSilicon) {
+    Seq("org.apache.spark.tags.ChromeUITest",
+      "org.apache.spark.tags.ExtendedLevelDBTest",
+      "org.apache.spark.tags.ExtendedRocksDBTest")
+  } else {
+    Seq("org.apache.spark.tags.ChromeUITest")
+  }
```

Comment:
How about:

```suggestion
  private val defaultExcludedTags = Seq("org.apache.spark.tags.ChromeUITest") ++
    if (isMacOnAppleSilicon) {
      Seq("org.apache.spark.tags.ExtendedLevelDBTest", "org.apache.spark.tags.ExtendedRocksDBTest")
    } else {
      Seq.empty
    }
```

rather than duplicate `ChromeUITest`?